package.json

```diff
@@ -103,12 +103,13 @@
     "core-js": "^2.4.0",
     "http-server": "^0.9.0",
     "ie-shim": "^0.1.0",
+    "marked": "^0.3.6",
     "nativescript-angular": "1.0.0",
     "nativescript-telerik-ui": "^1.3.1",
     "nativescript-theme-core": "0.0.9",
     "reflect-metadata": "^0.1.5",
     "rxjs": "5.0.0-beta.12",
-    "tns-core-modules": "2.3.0",
+    "tns-core-modules": "^2.3.0",
```

Comment:
Going to clean this one up... was a tweak of desperation when dealing with an errant android build issue.